runtests.py

```diff
@@ -1881,6 +1885,13 @@ def __call__(self, testname, tags=None):
                   % (testname, self._list_file))
         return exclude
 
+class CombinedExcluder(object):
```

Comment:
Doesn't seem used.

Reply:
No it isn't. I thought I deleted it when I decided not to use it, but clearly not. It'll go in the next round of revisions.